backend/main-service/src/main/java/kkakka/mainservice/product/domain/Product.java

```diff
@@ -1,4 +1,48 @@
 package kkakka.mainservice.product.domain;
 
+import kkakka.mainservice.category.domain.Category;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.ColumnDefault;
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name = "product")
+@Getter
+@NoArgsConstructor
 public class Product {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "PRODUCT_ID")
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    private Category category;
+
+    @Column(nullable = false)
+    private String name;
+    private Integer price;
+    private Integer stock;
+
+    @Column(nullable = true)
+    private String image_url;
+    private String detail_image_url;
+    private String nutrition_info_url;
```

Comment:
```suggestion
    @Column(nullable = true, name = "image_url")
    private String imageUrl;
    @Column(name = "detail_image_url")
    private String detail_imageUrl;
    @Column(name = "nutritionInfoUrl")
    private String nutritionInfoUrl;
```

프로덕션 코드에는 카멜케이스로 쓰고 DB 컬럼 네임을 지정해주면 더 좋을 것 같습니다!

Reply:
```suggestion
    @Column(nullable = true, name = "image_url")
    private String imageUrl;
    @Column(name = "detail_image_url")
    private String detailImageUrl;
    @Column(name = "nutrition_info_url")
    private String nutritionInfoUrl;
```

ㅎㅎㅎ 고치다가 좀 빼먹었는데 찰떡같은 이해 부탁드립니다....